package/Dockerfile_build_amd64

```diff
@@ -4,5 +4,17 @@ RUN apt-get update && apt-get install -y curl \
 
 COPY longhorn jivactl launch copy-binary launch-with-vm-backing-file launch-simple-jiva /usr/local/bin/
 
+ARG ARCH
+ARG DBUILD_DATE
+ARG DBUILD_REPO_URL
+ARG DBUILD_SITE_URL
+
+LABEL org.label-schema.schema-version="1.0"
```

Comment:
nit: should we also make version configurable?

Reply:
this is the schema version, not the product version.